### PR TITLE
Source

### PR DESCRIPTION
**Pull request by Karthik-M**
Added blog post on Apple watch connectivity.

### CODE REVIEW

File: source/_posts/2015-06-30-apple-watch-connectivity.markdown [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+categories: iOS
+---
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Apple Watch uses Wi-Fi and Bluetooth to communicate with your paired iPhone, switching between connections as needed:  
+
+* Apple Watch uses Bluetooth when your iPhone is near, which conserves power.  
+* If Bluetooth isn’t available, Apple Watch will try to use Wi-Fi. For example, if compatible Wi-Fi is available and your iPhone isn't in Bluetooth range,Apple Watch uses Wi-Fi.
+
+<center><img src="/images/posts/2015-06-30/Distance-between-iPhone-and-Apple-Watch.jpg" align="center" width="400" height="200" /></center>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;To enjoy every feature on your Apple Watch, you'll need to enable Wi-Fi and Bluetooth on your paired iPhone.Apple Watch compatible with Wi-Fi 802.11b/g and Bluetooth 4.0.
```
Comment (darcwader) [inline source/_posts/2015-06-30-apple-watch-connectivity.markdown]: this sentence is not required and ambiguous. what feature? enjoy?

File: source/_posts/2015-06-30-apple-watch-connectivity.markdown [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+
+### How do we communicate between watch and iPhone app ?
```
Comment (darcwader) [inline source/_posts/2015-06-30-apple-watch-connectivity.markdown]: maybe this should be the title of post

File: source/_posts/2015-06-30-apple-watch-connectivity.markdown [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The communication process between both devices is entirely different in watchOS 2 compared from watchOS 1. In this article, I am going to explain in detail, that how communication works in both OS versions.
+
+**<u>WatchOS 1:</u>**
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Basic communiation types are listed below:
```
Comment (darcwader) [inline source/_posts/2015-06-30-apple-watch-connectivity.markdown]: dont say below

File: source/_posts/2015-06-30-apple-watch-connectivity.markdown [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+
+<center><img src="/images/posts/2015-06-30/Distance-between-iPhone-and-Apple-Watch.jpg" align="center" width="400" height="200" /></center>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;To enjoy every feature on your Apple Watch, you'll need to enable Wi-Fi and Bluetooth on your paired iPhone.Apple Watch compatible with Wi-Fi 802.11b/g and Bluetooth 4.0.
+
+### How do we communicate between watch and iPhone app ?
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The communication process between both devices is entirely different in watchOS 2 compared from watchOS 1. In this article, I am going to explain in detail, that how communication works in both OS versions.
+
+**<u>WatchOS 1:</u>**
```
Comment (darcwader) [inline source/_posts/2015-06-30-apple-watch-connectivity.markdown]: use headings #, ##, ###, ####

**Comment by Karthik-M** [comment]
Done with the changes, please merge the code.

**Pull request merged**